zip-####.rst

```diff
@@ -155,62 +197,77 @@ SHOULD be applied to the Zcash Testnet for testing purposes.
 Dev Fund allocation
 -------------------
 
-Starting at the first Zcash halving in 2020, until the second halving in 2024,
-20% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
-consists of the following three slices:
+Starting at the second Zcash halving in 2024, until the third halving in 2028,
+15% of the block subsidy of each block SHALL be allocated to a "Dev Fund" that
+consists of the following allocations:
+
+* 5% for the Zcash Foundation (for internal work and grants);
+* 4% for the Bootstrap Project (the parent of the Electric Coin Company) for the 1st year only; 
+and contingent upon an ecosystem mandated "non-direct funding model" for the future;
+* 4% for Zcash Community Grants for continuation of their current activities as-is;
+* 2% for Qedit in support of their ZSA activities, and other Zcash protocol support;
 
-* 35% for the Bootstrap Project (denoted **BP slice**);
-* 25% for the Zcash Foundation, for general use (denoted **ZF slice**);
-* 40% for additional "Major Grants" for large-scale long-term projects
-  (denoted **MG slice**).
+Following the first year, when ECC will no longer receive their 4% allocation, 
+those block subsidies will be distributed as 1% to ZCG, 1% to ZecHub, 1% to FPF,
+and 1% to ZF.
 
-The slices are described in more detail below. The fund flow will be implemented
+This proposal mandates the ecosystem to design and deploy a "non-direct funding model"
+as generally recommended in Josh Swihart's proposal.
+<https://forum.zcashcommunity.com/t/zcash-funding-bloc-a-dev-fund-proposal-from-josh-at-ecc/47187>
```

Comment:
I'm confused at what is being proposed in this ZIP. It reads like it extends the direct funding model under different allocation & recipients, but then how does this fit with that? Is the idea that a non-direct funding model should be introduced as quickly as possible, and this ZIP would then be amended?

Reply:
Yes, good points/ questions.  My proposal doesn't anticipate that the creation of a Josh-like non-direct funding mechanism will happen in 1-year,  **but** it could be possible in less than 4-years.  So I'll add some exception handling, to be explicit about the scenario where the non-direct funding model is created in less than 4-years, which should trigger a re-visitation by the ecosystem as to how any further direct funding streams would exist (or not).  In the case of the non-direct funding model being completed, I would suggest that the community decides at that point in time whether or not direct funding should be continued or not.  This proposal recommends 4-years of direct funding, as detailed, regardless of how fast a non-direct funding model can be created/ activated, and would only cause this ZIP to be amended if ecosystem sentiment is a majority for a change to this ZIP's direction.